api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -428,3 +414,128 @@ def comment_incident(finance_incident_id: int) -> utils.BackofficeResponse:
             "backoffice_v3_web.finance_incident.get_incident", finance_incident_id=incident.id, active_tab="history"
         )
     )
+
+
+@finance_incident_blueprint.route("/incident/<int:finance_incident_id>/get-validation-form", methods=["GET"])
```

Comment:
On a tendance à plutôt mettre la même route en GET et POST pour récupérer le formulaire et le poster : 
```suggestion
@finance_incident_blueprint.route("/incident/<int:finance_incident_id>/validate", methods=["GET"])
```

Reply:
Si le méthodes HTTP et l'URL sont identiques, cela ne risque pas de créer un conflit entre ces deux endpoints ?

Reply:
Non tant que les `methods` ne se recoupent pas ; on le fait déjà par ailleurs.